crates/bevy_asset/src/path.rs

```diff
@@ -302,17 +335,24 @@ impl<'a> AssetPath<'a> {
         }
     }
 
+    /// Splits the internal path into components
+    #[inline]
+    pub fn path_components(&self) -> impl Iterator<Item = &str> {
+        self.path.split('/')
```

Comment:
We now use the separator in a bunch of different places. I was wondering whether we should add a constant for it? Or do you think too verbose. We wold probably need 2 constans for char and str.

Reply:
you're right, fixed